src/cultionet/__init__.py

```diff
@@ -1,5 +1,5 @@
 __path__: str = __import__('pkgutil').extend_path(__path__, __name__)
-__version__ = '1.6.5'
+__version__ = '1.7.0b'
```

Comment:
This will be changed to '1.7.0' after it's merged.

Reply:
will it be `1.7.0` or `1.7.0b`?

Reply:
Do you mean after we merge? 

> This will be changed to '1.7.0' after it's merged.

Maybe I misunderstood your question, but '1.7.0b' is what I have been working off of and we will release to '1.7.0'.

Reply:
> Do you mean after we merge?
yes, I mean after this PR merged.


 >  This will be changed to '1.7.0' after it's merged.

Ok, thanks for clarifying 👍